tests/model/RedirectorPageTest.php

```diff
@@ -48,18 +48,18 @@ public function testReflexiveAndTransitiveInternalRedirectors()
     {
         /* Reflexive redirectors are those that point to themselves.  They should behave the same as an empty redirector */
         $page = $this->objFromFixture(RedirectorPage::class, 'reflexive');
-        $this->assertEquals(Director::baseURL() . 'reflexive/', $page->Link());
+        $this->assertEquals('/reflexive/', $page->Link());
         $content = $this->get(Director::makeRelative($page->Link()))->getBody();
         $this->assertContains('message-setupWithoutRedirect', $content);
 
         /* Transitive redirectors are those that point to another redirector page.  They should send people to the URLSegment
 		 * of the destination page - the middle-stop, so to speak.  That should redirect to the final destination */
         $page = $this->objFromFixture(RedirectorPage::class, 'transitive');
-        $this->assertEquals(Director::baseURL() . 'good-internal/', $page->Link());
+        $this->assertEquals('/good-internal/', $page->Link());
 
         $this->autoFollowRedirection = false;
         $response = $this->get(Director::makeRelative($page->Link()));
-        $this->assertEquals(Director::baseURL() . "redirection-dest/", $response->getHeader("Location"));
+        $this->assertEquals(Director::absoluteURL('/redirection-dest/'), $response->getHeader("Location"));
```

Comment:
Answering my own question: the PR fixes the bug with absolute URL generation ;-)